sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -213,7 +213,7 @@ describe("Errors with non existing Queue/Topic/Subscription", async function():
     return namespace.close();
   });
 
-  const testError = (err: Error, entityPath: string) => {
+  const testError = (err: Error, entityPath: string): void => {
```

Comment:
@ramya-rao-a  what should be appropriate type for `testError` ? For this OnError doesn't look like the correct type

Reply:
void is right